site/outline/outline.rst

```diff
@@ -256,6 +251,42 @@ any questions they have regarding the material.
 
 
 
+Lecture & Lab Activities
+========================
+
+Lcture & lab activities will take during certain lectures and labs.
```

Comment:
typo _Lecture_

May be useful to pull up this text blob from git history, or store them in python or something and have rst stitch the files together (iunno if that is possible).

Reply:
I actually pulled it from the history of another, but changed it slightly which caused the typo. 

Good catch.